Shared/Flows/Onboarding/CodeViewController.swift

```diff
@@ -12,6 +12,8 @@ import Parse
 import TMROLocalization
 import Combine
 
+#warning("Move main actor to the base view controller.")
+@MainActor
```

Comment:
Ideally all the view controllers should be run on the Main Actor. Making that change is beyond the scope of this PR so I'm saving it for later.